src/app/user-auth/login-register/login-register.component.spec.ts

```diff
@@ -32,4 +32,4 @@
 
 //     expect(component.isClosed.emit).toHaveBeenCalledWith(true);
```

Comment:
Should not be commented out. Please before creating Pull Request make sure all commented code will not affect anything.

Reply:
I pulled from main before working today and it didn't change the code in my branch. Not sure as to why.